interview-questions.md

```diff
@@ -8,50 +7,69 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: An argument is what is passed through a function that will determine a specific output. A parameter is what is set within the function that determines what kind of data can be passed through the function.
```

Comment:
Incredible answers!  Really well researched and initial answers in this section!